k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageContainerView.java

```diff
@@ -413,6 +415,7 @@ public void onPageFinished() {
 
         if (!TextUtils.isEmpty(messageViewInfo.extraText)) {
             unsignedTextContainer.setVisibility(View.VISIBLE);
+            unsignedTextDivider.setVisibility(hideUnsignedTextDivider ? View.GONE : View.VISIBLE);
             unsignedText.setText(messageViewInfo.extraText);
```

Comment:
While we're at it, this data should also be cleared/View.GONE in `clearDisplayedContent`